### DISCUSSION TITLE
Do you support Flux? And ComfyUI?

### DISCUSSION THREAD

**Discussion by martjay** [shaitanzx/Fooocus_extend#5] [Ideas]
I hope you can support the interfaces of Flux and ComfyUI, as well as provide methods for customizing presets. We should turn it into an all - around tool. I think it's completely feasible. You can refer to SimpleSDXL for DIY. I can provide you with a reference project. They have implemented a lot of different model presets.

DOC：[feishu](https://acnmokx5gwds.feishu.cn/wiki/QK3LwOp2oiRRaTkFRhYcO4LonGe) 
Integrated Package：
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/SimpleAI.FluxAIO_dev.exe.7z1215
  Remove“1215”and unzip.
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/models_base_simpleai_1214.zip
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/models_enhance_simpleai_0908.zip
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/models_flux_aio_simpleai_1214.zip
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/models_kolors_aio_simpleai_1214.zip
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/models_sd15_aio_simpleai_1214.zip
https://huggingface.co/metercai/SimpleSDXL2/resolve/main/models_sd35_large_clips_simpleai_1214.zip
  And unzip all of model pack.

SimpleSDXL and SimpleModels are two different folders. When running SimpleSDXL, it will automatically locate the model directory.

GitHub：https://github.com/metercai/SimpleSDXL

The version on GitHub lags behind that on Feishu. Unfortunately, this person intends to turn the modified Fooocus into a paid item. I think this completely goes against the original intention of open - source software. So I hope you can replicate their branch project. By downloading their integrated package from Huggingface, we can get the latest version. However, you need to bind your phone number in the "Advanced" section first to access their advanced features.

**Comment by shaitanzx** [comment]
I would love to support Flux, but I don't have powerful enough hardware to do the testing.

**Reply by martjay** [reply]
> I would love to support Flux, but I don't have powerful enough hardware to do the testing.

Simplesdxl has made a lot of optimizations for flux, kolors, and SD3.5. They can run on even 6G video memory machines. Maybe you can learn from their code. Maybe they use some optimization methods. If I know how to code, I will also learn how to clone Simplesdxl. The method of optimizing performance and the method of the tab bar have been learned. I think that writing the gradio module in combination with the comfyui workflow can almost make a simplesdxl that is exactly the same, or even an infinite solution.
